doc/man1/ciphers.pod

```diff
@@ -756,6 +757,15 @@ Set security level to 2 and display all ciphers consistent with level 2:
 
  openssl ciphers -s -v 'ALL:@SECLEVEL=2'
 
+=head1 BUGS
+
+If there are extra characters after the word B<DEFAULT> before the separating
+colon, they are taken to be a cipher name.
```

Comment:
IIUC from comments by people who have read the code, they are taken as cipher string, not cipher name.  ie, `DEFAULT-ECDH` means what `DEFAULT:-ECDH` would, although `-ECDH` is not a cipher name.

Reply:
Yes, I guess so.  The terminology in this page is poor.